packager/signer/signer_test.go

```diff
@@ -380,18 +381,56 @@ func (this *SignerSuite) TestProxyUnsignedNonCachable() {
 	this.Assert().Equal("text/html", resp.Header.Get("Content-Type"))
 }
 
+func (this *SignerSuite) TestProxyUnsignedIfNotAMP() {
+	urlSets := []util.URLSet{{
+		Sign: &util.URLPattern{[]string{"https"}, "", this.httpsHost(), stringPtr("/amp/.*"), []string{}, stringPtr(""), false, nil}}}
+	nonAMPBody := []byte("<html><body>They like to OPINE. Get it? (Is he fir real? Yew gotta be kidding me.)")
+	this.fakeHandler = func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Naming: Can we use "writer" or something better than "w"? Optional as I know this code already does this a lot.

Reply:
OK, renamed to `resp` as that's pretty pervasive throughout the codebase.